packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx

```diff
@@ -316,6 +318,16 @@ export const CommonCallControlBar = (props: CommonCallControlBarProps & Containe
                         disableTooltip={props.mobileView}
                       />
                     )}
+                    {
+                      /* @conditional-compile-remove(reaction) */
+                      !props.mobileView && isEnabled(options.raiseHandButton) && (
```

Comment:
Can you explain this section? I see we add the reaction button below the RaiseHand also, shouldonly one of these exist?
Also I assume this isEnabled line should be isEnabled(options.reactionsButton) and not isEnabled(options.raiseHandButton)?
```suggestion
                      !props.mobileView && isEnabled(options.reactionsButton) && (
```

Reply:
Ah yeah looking at the snapshots, we should delete that second `<Reaction` usage below as there are two reaction buttons:
![image](https://github.com/Azure/communication-ui-library/assets/2684369/d9f275e1-8d9c-416c-b10c-60059280bb9c)

Reply:
Ops. This might have added due to merge conflicts. Removed the extra section. Thanks :)